zrml/swaps/src/lib.rs

```diff
@@ -7,6 +7,9 @@
 
 extern crate alloc;
 
+#[cfg(test)]
+#[macro_use]
+extern crate more_asserts;
```

Comment:
[nitpick] [`#[macro_use]`](https://doc.rust-lang.org/1.29.0/book/first-edition/macros.html#scoping-and-macro-importexport) leads to a global macro import, which can be useful when having tests in multiple different modules that use those macros. Otherwise the import is too broad and it should be used only within the relevant module (by using the `use` statement), like the fixed module in our case. 
This is just my opinion. For example, by using this approach executing specific tests (e.g. `cargo test fixed`) will import the macros, whereas other tests won't.

Reply:
Done!